### DISCUSSION TITLE
AVIF Q vs JPEG Q?

### DISCUSSION THREAD

**Discussion by zhifengzhuang** [libvips/libvips#3246] [Q&A] [answered]
hello, @jcupitt, vips_heifsave has a Q argument.Q 30 gives about the same quality as JPEG Q 75. When JPEG Q is 100, what should we set AVIF Q?Is there any relationship between JPEG Q and AVIF Q?

**Comment by jcupitt** [comment, marked as answer]
Hi @zhifengzhuang,

No, there's no simple relationship -- Q is just a rough guide. I would set AVIF Q to 100 as well.

**Reply by zhifengzhuang** [reply]
So, AVIF also has Q between 0 and 100?It's weird that AVIF Q 30 equals to JPEG 75😂.

**Reply by zhifengzhuang** [reply]
If I want to keep quality when parse jpeg to avif.How can I determinate Q of AVIF?It's there any good idea?

**Reply by jcupitt** [reply]
Maybe something like:

```
avif_q = min(100, jpeg_q * 2)
```

It won't be exact, of course.

JPEG Q is different between different jpeg libraries, so you won't find a match between two different image formats.

**Reply by zhifengzhuang** [reply]
Ok, thanks @jcupitt .
